<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="MyQCustomPlot" name="customPlot" native="true">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadDataFile"/>
    <addaction name="actionReloadDataFile"/>
    <addaction name="separator"/>
    <addaction name="actionExportImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionWatchFile"/>
    <addaction name="actionDynamicSession"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuScale">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Scale</string>
     </property>
     <addaction name="actionAutoScaleXAxis"/>
     <addaction name="actionAutoScaleYAxis"/>
     <addaction name="separator"/>
     <addaction name="actionSetManualScaleXAxis"/>
     <addaction name="actionSetManualScaleYAxis"/>
    </widget>
    <widget class="QMenu" name="menuShowHide">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Show/Hide</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuBringToFront">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Bring to front</string>
     </property>
    </widget>
    <addaction name="menuScale"/>
    <addaction name="separator"/>
    <addaction name="menuShowHide"/>
    <addaction name="menuBringToFront"/>
    <addaction name="separator"/>
    <addaction name="actionHighlightSamplePoints"/>
    <addaction name="separator"/>
    <addaction name="actionClearMarkers"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGraph"/>
   <addaction name="menuView"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QDockWidget" name="markerInfoDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Markers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="MarkerInfo" name="markerInfo">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>432</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="legendDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Legend</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="VerticalScrollAreaContents" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>69</width>
          <height>77</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="Legend" name="legend">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionLoadDataFile">
   <property name="text">
    <string>Load Data File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReloadDataFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reload Data File</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionExportImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Image</string>
   </property>
  </action>
  <action name="actionAutoScaleXAxis">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto scale x-axis</string>
   </property>
  </action>
  <action name="actionAutoScaleYAxis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Auto scale y-axis</string>
   </property>
  </action>
  <action name="actionSetManualScaleXAxis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set Manual scale x-axis...</string>
   </property>
  </action>
  <action name="actionSetManualScaleYAxis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set Manual scale y-axis...</string>
   </property>
  </action>
  <action name="actionWatchFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Watch File</string>
   </property>
  </action>
  <action name="actionDynamicSession">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dynamic Session</string>
   </property>
  </action>
  <action name="actionHighlightSamplePoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highlight Sample Points</string>
   </property>
  </action>
  <action name="actionClearMarkers">
   <property name="text">
    <string>Clear Markers</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Legend</class>
   <extends>QFrame</extends>
   <header>legend.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VerticalScrollAreaContents</class>
   <extends>QWidget</extends>
   <header>verticalscrollareacontents.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MarkerInfo</class>
   <extends>QFrame</extends>
   <header>markerinfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyQCustomPlot</class>
   <extends>QWidget</extends>
   <header>myqcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
